apple/internal/experimental_mixed_language_library.bzl

```diff
@@ -341,7 +341,7 @@ target only contains Objective-C files.""")
             # attribute to declare custom inputs.
             ":" + umbrella_module_map,
         ],
-        module_map = umbrella_module_map,
+        module_map = ":" + umbrella_module_map,
```

Comment:
Since we do this above as well, can it get pulled out so we only do the string creation once?